.github/update_projects_json.py

```diff
@@ -0,0 +1,35 @@
+from pathlib import Path
+from spacy.cli._util import PROJECT_FILE, load_project_config
+from wasabi import msg
+import json
+import typer
+
+
+def main(root: Path = typer.Argument(Path.cwd(), help="Root path to look in")):
+    """
+    Update the projects.json file for the repo.
+
+    Unlike the docs update script, this is desigend to only be run on the root
+    of the whole repository.
+    """
+    msg.info(f"Updating projects.json in {root}")
+    entries = []
+    # We look specifically for project directories
+    for path in root.glob(f"**/*/{PROJECT_FILE}"):
+        path = path.parent
+
+        # prep data for the json file
+        config = load_project_config(path)
+        entry = {"shortname": f"{path.parent.name}/{path.name}"}
+        entry["title"] = config["title"]
```

Comment:
Can we guarantee that `title` will always be available?

Reply:
In the schema the title is technically optional, so in general, we can't guarantee it's there. However, for this specific repo, I would consider it an error for any project to not have a title.